samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/author/CalculatedQuestionGlobalVariableBean.java

```diff
@@ -0,0 +1,134 @@
+/**********************************************************************************
+ * $URL$

```

Comment:
Let's use the new header!

https://github.com/sakaiproject/sakai/blob/master/ECLv2-header-template.txt

Reply:
New header added!